### PR TITLE
Create Kadane_algorithm.cpp

### PR DESCRIPTION
**Pull request by mayankmj**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

**Comment by mayankmj** [comment]
just a request pls merge!! today is the last date

File: C++/Kadane_algorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    int arr[n];
+    array_input(n,arr);
+    int maximum=0,currrent=0;
+    for(int i=0;i<n;i++)
+    {
+        current=current+arr[i];
+        if(current>maximum)
+        maximum=current;
+        if(current<0)
+        current=0;
```
Comment (aravindvnair99) [inline C++/Kadane_algorithm.cpp:39]: You need to declare `current` first before using it. Will lead to compilation error.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
> just a request pls merge!! today is the last date

@mayankmj Just a tip, last-minute is never good and patience is a virtue. :)

Please resolve the errors in your code and your PR shall be merged.

**Comment by aravindvnair99** [comment]
Duplicate of #32

**Pull request closed without merging**